src/core.js

```diff
@@ -400,27 +410,33 @@ define([
         });
     };
 
+    fn.toggleFullScreen = function () {
+        var _this = this,
+            $fullScreenMenuItem = $(_.find(_this.$f.find('.fd-tools-menu').nextAll(), function(li) {
+                return $(li).find("a").text().match(/editor/i);
```

Comment:
This seems pretty fragile since it will pick up any other menu item that happens to contain the word "editor" Suggestion: change to `/(shrink|expand) editor/i`